Automattic/jetpack
# Boost: Minor CSS changes to Purchase Successful page

Fixes #29540

## Proposed changes:

* Update checklist spacing and button font size and line height.

### Other information:


## Jetpack product discussion



https://github.com/Automattic/jetpack/issues/29540

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup this version of Boost;
* Open `/wp-admin/admin.php?page=jetpack-boost/#purchase-successful` and see the changes.
